_layouts/base.html.haml

```diff
@@ -24,6 +24,7 @@ change_frequency: monthly
     %link(rel="stylesheet"){:href => relative("/website/highlight/highlight.css")}
     %link(rel="stylesheet"){:href => relative("/website/tabzilla/tabzilla.css")}
     %link(rel="stylesheet"){:href => relative("/website/optaplannerWebsite.css")}
+    %link(rel="stylesheet"){:href => "https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css"}
```

Comment:
Ok, so let's not use icons at all for now?
For now we can use no color coding or just color code the words IMPL_DETAIL, MINOR, MAJOR etc with a forerground color of yellow, orange and red. Maybe purple for README and blue for RECOMMENDED? We can tweak the actual colors later on.